File: doc/src/scripts/OrbitalSpeed.py
import time
import krpc

conn = krpc.connect(name='Orbital speed')
vessel = conn.space_center.active_vessel
ref_frame = vessel.orbit.body.non_rotating_reference_frame

while True:
    velocity = vessel.flight(ref_frame).velocity
    print('Orbital velocity = (%.1f, %.1f, %.1f)' % velocity)

    speed = vessel.flight(ref_frame).speed
    print('Orbital speed = %.1f m/s' % speed)

    time.sleep(1)

File: doc/src/scripts/OrbitalSpeed.lua
local krpc = require 'krpc'
local conn = krpc.connect('Orbital speed')
local vessel = conn.space_center.active_vessel
local ref_frame = vessel.orbit.body.non_rotating_reference_frame

while true do
    velocity = vessel:flight(ref_frame).velocity
    print('Orbital velocity =', velocity[1], velocity[2], velocity[3])

    speed = vessel:flight(ref_frame).speed
    print('Orbital speed = ' .. speed .. ' m/s')

    krpc.platform.sleep(1)
end
